### PR TITLE
[regression test](broker load) add case for without filepath

### PR DESCRIPTION
**Pull request by liugddx**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by liugddx** [comment]
run buildall

**Review by dataroaring — DISMISSED** [review]
LGTM

File: regression-test/suites/load_p0/broker_load/test_broker_load_without_filepath.groovy [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+                        (1,2023-09-01,1,1,1)
+                        """
+        assertTrue(result2.size() == 1)
+        assertTrue(result2[0].size() == 1)
+        assertTrue(result2[0][0] == 1, "Insert should update 1 rows")
+    }
+
+    def load_from_hdfs_norm = {testTablex, label, hdfsFilePath, format, brokerName, hdfsUser, hdfsPasswd ->
+        try {
+            sql """
```
Comment (morningman) [inline regression-test/suites/load_p0/broker_load/test_broker_load_without_filepath.groovy:56]: 2 issues:
1. If this sql does not throw exception, than your `assertTrue` will not be executed.
2. for exception test, you can just use `test` action, eg:
```
test {
    sql "xxxx";
    exception "your excepted error message"
}
```

**Comment by liugddx** [comment]
run buildall

**Review by morningman — APPROVED** [review]
LGTM

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**